plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -16,9 +16,9 @@
   private IProject project;
 
   public StandardDeployJob(ProjectToStagingExporter exporter,
-                        StagingDirectoryProvider stagingDirectoryProvider,
-                        IProject project) {
-    super(Messages.getString("deploy.standard.runnable.name"));
+                           StagingDirectoryProvider stagingDirectoryProvider,
```

Comment:
It seems strange to me that we'd pass in a StagingDirectoryProvider here rather than obtaining the staging directory and configuring the job with it.  I could see a provider being passed in if we *might* need the directory